release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -102,6 +102,21 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-images"]
 === Images
 
+[id="ocp-4-13-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-13-psa-tp-feature-set"]
+==== Pod security admission restricted enforcement (Technology Preview)
+
+With this release, pod security admission restricted _enforcement_ is available as a Technology Preview feature by enabling the `TechPreviewNoUpgrade` feature set. If you enable the `TechPreviewNoUpgrade` feature set, pods are rejected if they violate pod security standards, instead of only logging a warning.
+
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
```

Comment:
Consider using this note, which is used in a few places in `main`, to avoid "should:"

> Enabling the `TechPreviewNoUpgrade` feature set cannot be undone and prevents minor version updates. These feature sets are not recommended on production clusters.

Reply:
@kalexand-rh This exact warning is used many other places, so I don't want to change the wording here. Probably should be a snippet honestly.

Funny enough, I tried to recommend the same thing during a peer review involving this note elsewhere (to avoid "should") and had to backtrack because of the thought that changing it might need change management, because the meaning could be different. :shrug: